idb/mocks/IndexerDb.go

```diff
@@ -6,15 +6,13 @@ import (
 	context "context"
 
 	idb "github.com/algorand/indexer/idb"
-	indexertypes "github.com/algorand/indexer/types"
-
-	ledgercore "github.com/algorand/go-algorand/ledger/ledgercore"
-
 	mock "github.com/stretchr/testify/mock"
 
 	testing "testing"
 
-	types "github.com/algorand/go-algorand-sdk/v2/types"
+	types "github.com/algorand/indexer/types"
+
+	v2types "github.com/algorand/go-algorand-sdk/v2/types"
```

Comment:
nit: I guess I would personally have made this `sdk "github.com/algorand/go-algorand-sdk/v2/types"`

Reply:
unfortunately, this is auto-generated by mockery.